ee/clickhouse/views/test/test_clickhouse_experiments.py

```diff
@@ -681,30 +681,13 @@ def test_invalid_create(self):
                 "end_date": None,
```

Comment:
@jurajmajerik Ah, I missed this file. Should the tests I created live here instead?

Also, out of curiosity, what's the reason 'clickhouse' is included in the file name?

Reply:
> Should the tests I created live here instead?

Ah good point. Yes indeed - do you mind moving them?

Reply:
> Yes indeed - do you mind moving them?

Yep, done in ea46fe8f4d

Reply:
a long time ago, `ee` was very roughly for clickhouse backed stuff, and `posthog/` was for postgres backed stuff (when we had analytics data in postgres, and no support for open source clickhouse). Experiments were clickhouse only from the beginning, and thus put in ee like so.